pkg/operator/ceph/csi/version_test.go

```diff
@@ -23,14 +23,16 @@ import (
 )
 
 var (
-	testMinVersion  = CephCSIVersion{3, 7, 0}
+	testMinVersion  = CephCSIVersion{3, 8, 0}
 	testReleaseV350 = CephCSIVersion{3, 5, 0}
```

Comment:
Can we delete these definitions for 3.5, 3.6, and 3.7? They aren't used anymore, right?

Reply:
removed 3.5 and 3.6 and nscenter and customcephconfig checks since its not required anymore. 
both are support from 3.8 version